spotify/config.yaml

```diff
@@ -15,9 +15,11 @@ init: false
 options:
   name: Home Assistant
   bitrate: 160
+  autoplay: true
```

Comment:
```suggestion
```

I would not add this as a default option, it would change functionality for existing users.

Reply:
Sure, I'll remove that